.github/scripts/liboboe-checksum.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# input is a space delimited string of files to checksum
+# e.g. "liboboe-1.0-x86_64.so.0.0.0 liboboe-1.0-alpine-x86_64.so.0.0.0 liboboe-1.0-lambda-x86_64.so.0.0.0"
+# we assume each file has a matching .sha256 file.
+files="$1"
+
+# hard coded location of sha files
+url="https://files.appoptics.com/c-lib"
+
+errors=0
+error_files=""
+
+for f in $files; do
+    # get data from binary file and sha256 file
+    version=$(cat ./oboe/VERSION)
+    echo "Downloading ${url}/${version}/$f.sha256"
+    curl -o "./oboe/${f}.sha256" "${url}/${version}/$f.sha256"
+
+    correct=$(awk '{print $1}' < "./oboe/$f.sha256")
+    checked=$(sha256sum "./oboe/$f" | awk '{print $1}')
```

Comment:
Minor: I'd find it clearer to call the correct/checked pair something like published/vendored or downloaded/local instead.  We hope the downloaded is the correct one but that may not be the case if our fileserver was actually tampered with.